src/libs/actions/Task.ts

```diff
@@ -317,11 +346,11 @@ function reopenTask(taskReport) {
         {
             onyxMethod: Onyx.METHOD.MERGE,
             key: `${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${taskReportID}`,
-            value: {[reopenedTaskReportAction.reportActionID]: reopenedTaskReportAction},
+            value: {[reopenedTaskReportAction.reportActionID]: reopenedTaskReportAction as OnyxTypes.ReportAction},
```

Comment:
Why there are so many ReportAction assertion in this PR? 🤔 Maybe something is wrongly typed?

Reply:
So the problem here is that OnyxUpdate expect that `value` always will be for example of `ReportAction` type but in those cases we are passing just an optimistic updates and those dont have all required props in `ReportAction` type. For example `reopenedTaskReportAction` const is of type `OptimisticTaskReportAction` which looks like that 

type OptimisticTaskReportAction = Pick<
    ReportAction,
    | 'reportActionID'
    | 'actionName'
    | 'actorAccountID'
    | 'automatic'
    | 'avatar'
    | 'created'
    | 'isAttachment'
    | 'message'
    | 'originalMessage'
    | 'person'
    | 'pendingAction'
    | 'shouldShow'
    | 'isFirstItem'
    | 'previousMessage'
    | 'errors'
    | 'linkMetadata'
>;